deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CodegenConfig.java

```diff
@@ -82,6 +83,10 @@ public static String getImportMappingsPropertyName(final Path openApiFilePath) {
         return String.format(IMPORT_MAPPINGS_PROP_FORMAT, getBuildTimeSpecPropertyPrefix(openApiFilePath));
     }
 
+    public static String getCodegenLanguagePropertyName(final Path openApiFilePath) {
```

Comment:
@ricardozanini where would you put the switch between languages?

Reply:
I like it being on an individual API-spec level! (instead of it being set globally for a project) That makes the interop with java nicer and makes this useable in mixed kotlin/java projects

Reply:
Yes, I think this is important, because many projects are mixed!
I also thought about kotlin being the default when `quarkus-kotlin` extension is present and java being default when not. What do you think?

Reply:
That makes sense to me. :)